lib/stats-collector.js

```diff
@@ -1,19 +1,21 @@
 'use strict';
 
 /**
- * Initializes test statistics collector object.
+ * Creates test statistics collector object.
  *
  * Provides stats such as test duration,
  * number of tests passed / failed etc.
  *
  * @public
  * @memberof Mocha
  * @param {Runner} runner
- * @return {StatsCollector}
- *
- * Reporter statistics.
+ * @return {stats}
+ */
+
+/**
+ * {stats} object properties
```

Comment:
```text
 * Test statistics
```

Reply:
@plroebuck  sorry I don't follow this comment, it is not a `Test` right?

Reply:
[Like this](https://github.com/mochajs/mocha/blob/fe623c745c44085ee79b1978e98969454d662848/lib/stats-collector.js#L4):
```js
/**
 * Test statistics collector.
 *
 * @typedef {Object} StatsCollector
...
```

Reply:
My query is around the term "Test". What is this module testing?